posthog/api/decide.py

```diff
@@ -215,7 +215,8 @@ def get_decide(request: HttpRequest):
             )
 
             if settings.NEW_ANALYTICS_CAPTURE_TEAM_IDS and str(team.id) in settings.NEW_ANALYTICS_CAPTURE_TEAM_IDS:
-                response["analytics"] = {"endpoint": settings.NEW_ANALYTICS_CAPTURE_ENDPOINT}
+                if random() < settings.NEW_ANALYTICS_CAPTURE_SAMPLING_RATE:
```

Comment:
one test here to ensure this works fine would be really nice. You can follow this one: https://github.com/PostHog/posthog/blob/master/posthog/api/test/test_decide.py#L2203 for how to override these settings in test

Reply:
Thanks, I'm indeed writing a test right now. Trying to find out the best way to assert that `response.json()["analytics"]` does not exist

Reply:
`"analytics" not in response.json()` ?